content/it/docs/concepts/overview/components.md

```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
 
 ### cloud-controller-manager
 
-Il [cloud-controller-manager](/docs/tasks/administer-cluster/running-cloud-controller/) esegue i controller che interagiscono con i cloud provider responsabili per la gestione dell'infrastruttura sottostante al cluster, in caso di deployment in cloud.
-Il cloud-controller-manager è una funzionalità alpha introdotta in Kubernetes 1.6.
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
 
-Il cloud-controller-manager esegue esclusivamente i cicli di controllo specifici dei cloud provider.
-È possibile disabilitare questi cicli di controllo usando il kube-controller-manager.
-È inoltre possibile disabilitare i cicli di controllo settando il parametro `--cloud-provider` con il valore `external` durante l'esecuzione del kube-controller-manager.
+Il cloud-controller-manager esegue controller specifiche del tuo cloud provider.
+Se hai una installazione Kubernetes on premises, o un ambiente di laboratorio
+nel tuo PC, il cluster non ha un cloud controller manager.
 
-Il cloud-controller-manager permette l'evoluzione indipendente al codice di Kubernetes e a quello dei singoli cloud vendor.
-Precedentemente, il codice core di Kubernetes dipendeva da implementazioni specifiche dei cloud provider.
-In futuro, implementazioni specifiche per singoli cloud provider devono essere mantenuti dai cloud provider interessati e collegati al cloud-controller-manager.
+Come nel kube-controller-manager, il cloud-controller-manager combina diversi control loops 
+logicamente indipendenti in un singolo binario che tu puoi eseguire come un singolo processo. Tu puoi
+scalare orizzontalmente ( eseguire più di una copia ) per migliorare la responsività o per migliorare la tolleranza ai fallimenti.
 
 I seguenti controller hanno dipendenze verso implementazioni di specifici cloud provider:
 
   * Node Controller: Per controllare se sul cloud provider i nodi che hanno smesso di rispondere sono stati cancellati
-  * Route Controller: Per configurare le regole di route nella sottostante infrastruttura cloud
-  * Service Controller: Per creare, aggiornare ed eliminare i load balancer nella infrastruttura del cloud provider
-  * Volume Controller: Per creare, associare e montare i volumi e per interagire con il cloud provider per orchestrare i volumi
-
+  * Route Controller: Per configurare le rotte nella sottostante infrastruttura cloud
```

Comment:
I think "rotte" is not very clear in Italian. Though, I'm not really sure how it could be written in a more clear way

Reply:
If you are a nteworking guy, "rotte" is a popular term, but I think we should think about to use "route" as term.

Reply:
I will allign rotte like the suggestion Fabrizio did with "network routes"